src/SearchInput/SearchInput.js

```diff
@@ -207,6 +219,8 @@ SearchInput.propTypes = {
     className: PropTypes.string,
     /** Set to **true** to enable compact mode */
     compact: PropTypes.bool,
+    /** Internal use only */
+    disableStyles: PropTypes.bool,
```

Comment:
`disableStyles` is not needed anymore: https://github.com/SAP/fundamental-react/pull/1102

Reply:
Yes I have removed it.